catalog-rest-service/src/main/java/org/openmetadata/catalog/util/EntityUtil.java

```diff
@@ -161,6 +165,17 @@ public static EntityReference getService(EntityRelationshipDAO dao, String entit
     return refs.isEmpty() ? null : refs.get(0);
   }
 
+  public static EntityReference getService(
+      EntityRelationshipDAO dao, String entityType, UUID entityId, Include include) {
+    List<EntityReference> refs =
+        dao.findFrom(entityId.toString(), entityType, Relationship.CONTAINS.ordinal(), toBoolean(include));
+    if (refs.size() > 1) {
+      LOG.warn("Possible database issues - multiple services found for entity {}", entityId);
+      return refs.get(0);
```

Comment:
Delete this return statement and just log more than 1 results as a warning

Reply:
Ok.

Reply:
Done.